{
    "type": "bundle",
    "id": "bundle--7ec9e1a8-dfd0-40b9-8f98-47f05f03afa3",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--618f7a38-0a15-4b4c-917e-55ee86ace125",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-05-21T00:43:05.194235Z",
            "modified": "2022-05-21T00:43:05.194235Z",
            "name": "CVE-2022-29193",
            "description": "TensorFlow is an open source platform for machine learning. Prior to versions 2.9.0, 2.8.1, 2.7.2, and 2.6.4, the implementation of `tf.raw_ops.TensorSummaryV2` does not fully validate the input arguments. This results in a `CHECK`-failure which can be used to trigger a denial of service attack. Versions 2.9.0, 2.8.1, 2.7.2, and 2.6.4 contain a patch for this issue.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-29193"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}